README.adoc

```diff
@@ -186,10 +189,10 @@ broadcasted, or sent to all registered sinks, by invoking the
 [hotspot=broadcastEvent file=0]`SseBroadcaster.broadcast()` method.
 
 
-You have now set up an endpoint in the `bff` service that the front-end client
-will use to subscribe to events. You have also enabled the service to read from
-the reactive messaging channel and broadcast the information as
-events to subscribers via SSE.
+You have now set up an endpoint in the `bff` service that the client in the
```

Comment:
**change**
You have now set up an endpoint in the `bff` service that the client in the
`frontend` service will use to subscribe to events. You have also enabled the
service to read from the reactive messaging channel and broadcast the
information as events to subscribers via SSE.

**to**
You just set up an endpoint in the `bff` service that the client in the
`frontend` service can use to subscribe to events. You also enabled the
service to read from the reactive messaging channel and broadcast the
information as events to subscribers via SSE.